pkgs/by-name/fl/flatpak/package.nix

```diff
@@ -175,37 +188,42 @@ stdenv.mkDerivation (finalAttrs: {
     "installed_test_metadir=${placeholder "installedTests"}/share/installed-tests/flatpak"
   ];
 
-  postPatch = let
-    vsc-py = python3.withPackages (pp: [
-      pp.pyparsing
-    ]);
-  in ''
-    patchShebangs buildutil
-    patchShebangs tests
-    PATH=${lib.makeBinPath [vsc-py]}:$PATH patchShebangs --build subprojects/variant-schema-compiler/variant-schema-compiler
-  '';
+  postPatch =
+    let
+      vsc-py = python3.withPackages (pp: [ pp.pyparsing ]);
+    in
+    ''
+      patchShebangs buildutil
+      patchShebangs tests
+      PATH=${lib.makeBinPath [ vsc-py ]}:$PATH patchShebangs --build subprojects/variant-schema-compiler/variant-schema-compiler
+    '';
 
   passthru = {
     icon-validator-patch = substituteAll {
       src = ./fix-icon-validation.patch;
       inherit (builtins) storeDir;
     };
 
+    updateScript = nix-update-script { };
+
     tests = {
       installedTests = nixosTests.installed-tests.flatpak;
 
       validate-icon = runCommand "test-icon-validation" { } ''
         ${finalAttrs.finalPackage}/libexec/flatpak-validate-icon --sandbox 512 512 ${../../../applications/audio/zynaddsubfx/ZynLogo.svg} > "$out"
```

Comment:
Maybe we can try switching to `"${nixos-icons}/share/icons/hicolor/scalable/apps/nix-snowflake-white.svg"` to fix the check.